letsencrypt-win-simple/Plugin/OverviewPlugin.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Serilog;
+using Microsoft.Web.Administration;
+
+namespace LetsEncrypt.ACME.Simple
+{
+    public class OverviewPlugin : Plugin
+    {
+        private const string ClientName = "letsencrypt-win-simple";
+
+        public override string Name => "OverviewPlugin";
+
+        public override void PrintMenu()
+        {
+            Console.WriteLine(" O: Get overview of all bindings, protocols, renewals and certificates");
+        }
+
+        public override void HandleMenuResponse(string response, List<Target> targets)
+        {
+            if (response == "o")
+            {
+                Console.WriteLine("Running Overview Plugin");
+                Log.Information("Running Overview Plugin");
+
+                GetIisSites();
+                Console.WriteLine();
+                Console.WriteLine("Press enter to continue.");
+                Console.ReadKey();
+                GetRenewals();
+                Console.WriteLine();
+                Console.WriteLine("Press enter to continue.");
+                Console.ReadKey();
+                GetCertificates();
+                Console.WriteLine();
+            }
+        }
+
+        public static void GetIisSites()
+        {
+            using (var iisManager = new ServerManager())
+            {
+                Console.WriteLine("IIS Sites:");
+                foreach (var site in iisManager.Sites)
+                {
+                    Console.WriteLine("  " + site.Id + ": " + site.Name);
+                    foreach (var binding in site.Bindings.OrderBy(o => o.Host))
+                    {
+                        if (!String.IsNullOrEmpty(binding.Host))
+                        {
+                            var line = "     - " + binding.Host + ((binding.Protocol == "https") ? " (https)" : "");
+                            Console.WriteLine(line);
+                        }
+                    }
+                }
+            }
+        }
+
+        public static void GetCertificates()
+        {
+            try
+            {
+                Console.WriteLine();
+                Console.WriteLine("Certificates:");
+                string _certificateStore = "WebHosting";
```

Comment:
Yep, it shouldn't have that hard coded in the plugin as the value is already set in the main program and it can be different. I'll give an example when I add in all of my PR comments.